src/http/StateFlags.h

```diff
@@ -16,7 +16,7 @@ class StateFlags
 {
 public:
     unsigned int front_end_https = 0; ///< send "Front-End-Https: On" header (off/on/auto=2)
-    bool keepalive = false;
+    bool connectionClose = false; ///< send Connection: close header
```

Comment:
Why rename this data member? Renaming makes us out of sync with other "keep-alive"-based names. Are we changing the data member meaning?